.env.example

```diff
@@ -1,2 +1,2 @@
-GATSBY_DEFAULT_SITE_URL=http://localhost
+GATSBY_DEFAULT_SITE_URL=http://localhost:8000
```

Comment:
It should not be a required field. So you can run the project without filling it.